src/module_loader.rs

```diff
@@ -89,18 +92,76 @@ impl InnerRustyLoader {
     }
 }
 
+pub trait ImportProvider {
+    fn resolve(
+        &self,
+        url: &str,
+        referrer: &str,
+        _import_kind: deno_core::ResolutionKind,
+    ) -> Result<Url, anyhow::Error> {
+        match deno_core::resolve_import(url, referrer) {

```

Comment:
This doesn't really need a default implementation

Reply:
I figured it'd be useful for those who don't need to change that method. I wasn't planning to use non-default behavior for that in my use case anyway. I just need custom import schemes.